src/test/java/org/openstreetmap/atlas/geography/atlas/items/SnappedLineItemTest.java

```diff
@@ -0,0 +1,57 @@
+package org.openstreetmap.atlas.geography.atlas.items;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.geography.Location;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.dynamic.rules.DynamicAtlasTestRule;
+import org.openstreetmap.atlas.utilities.scalars.Distance;
+
+/**
+ * @author chunzc
+ */
+public class SnappedLineItemTest
+{
+    @Rule
+    public final DynamicAtlasTestRule rule = new DynamicAtlasTestRule();
```

Comment:
Here it might be nice to have its own rule, even if it is just a copy. In this case if someone comes in later and refactors the DynamicAtlas rule to adapt it to a new requirement for example, it won't impact that specific test here.